Makefile

```diff
@@ -16,10 +18,24 @@ cook-image:
 push-image:
 	docker push $(DOCKER_TAG)
 
-.PHONY: test
-test:
-	tox
+venv: requirements.txt
+	python ./vendor/venv-update venv= venv -ppython3 install= -r requirements.txt
+
+.PHONY: install-hooks
+install-hooks: venv
+	$(BIN)/pre-commit install -f --install-hooks
+
+test: lint
```

Comment:
This is still `.PHONY`, right? Are there going to be future things added to this? If not, I'd just remove the `lint` target and only use `test` instead, as it's the standard one to use.